ui/index.html

```diff
@@ -7,8 +7,7 @@
     <link rel="stylesheet" href="./src/index.css" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Go-Fast CDN</title>
-  </head>
-  <body class="min-w-full min-h-screen overflow-hidden">
+  </head>  <body class="min-w-full min-h-screen">
```

Comment:
```suggestion
  </head>  
  <body class="min-w-full min-h-screen">
```
Formatting seems broken